OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -215,6 +215,31 @@ function Get-TargetResource {
     }
 }
 
+function Test-ValidJson
+{
+    param($string)
+    try {
+        $string | ConvertFrom-Json
+        return $true
+    }
+    catch {
+        return $false
+    }
+}
+
+function Get-CleanedJson
+{
+    param($jsonstring)
+    $jsonstart = $jsonstring.IndexOf("{")
+    Write-Verbose "Found start of JSON at character $jsonstart"
+    $extractedjson = $jsonstring.Substring($jsonstart, $jsonstring.length - $jsonstart)
+
+    $dumpedstring = $jsonstring.substring(0, $jsonstart-1)
+    Write-Verbose "stripped extra content from JSON configuration string`r`n`r`n$dumpedstring"
```

Comment:
This feels like it should be `write-warning`

Reply:
I changed it from write waring in a fit of self-doubt. Happy to change it back if you are.